mongoConnection.ts

```diff
@@ -0,0 +1,19 @@
+import { MongoClient } from "mongodb";
+import * as dotenv from "dotenv";
+
+// was having an issue with the types, we can fix this later...
+let _connection: any = undefined;
+let _db: any = undefined;
```

Comment:
```suggestion
import { MongoClient, Db } from "mongodb";
import * as dotenv from "dotenv";

let _connection: MongoClient | null = null;
let _db: Db | null = null;
```

Reply:
To add on further you could change the first line of David's suggestion to `import type { ...`

Reply:
Just curious @carpenterd777 why the switch to null instead of undefined? Seems to me like this would be simpler:
`let _connection: MongoClient? = undefined;` or even `let _connection: MongoClient?`

Reply:
ah that probably makes more sense, I switched it to `null` because I thought that undefined was typically reserved for errors. But I guess not being able to connect to the db would be a pretty big error

Reply:
Ohhhhhh I must have been tired when trying to fix the typeerror I was getting...I just needed to remove the `= undefined` from these....thank you!!! It was so obvious, too!